content/20.treatments.md

```diff
@@ -540,19 +540,60 @@ Although exciting, the cost of vaccine manufacturing and electroporation may mak
 
 #### RNA Vaccines
 
-Brief background on the therapeutic.
+RNA vaccines are nucleic-acid based modalities that code for viral antigens against which our body elicits a humoral and cellular immune response. 
+The mRNA technology is transcribed in vitro and delivered to cells via lipid nanoparticles (LNP). 
+They are recognized by ribosomes in vivo and then translated and modified into functional proteins [@https://doi.org/10.3389/fimmu.2019.00594]. 
```

Comment:
```suggestion
They are recognized by ribosomes in vivo and then translated and modified into functional proteins [@doi:10.3389/fimmu.2019.00594]. 
```
note the `@doi:` syntax that you used below works while this one using the full URL (`@https:`) will not.